sysvinit/gentoo/cloud-init-hotplug

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+
```

Comment:
I don't have much depth in openrc files, so please excuse the questions if they don't make sense.

It seems given other examples of openrc files, that we are missing helpful attributes that allow others to introspect this daemon's purpose, looking over /etc/init.d/sshd I see we can have helpful attributes such as 
```suggestion
description="cloud-init hotplug hook daemon"
```

Without the above diff I can't see much about this daemon
```
~ # rc-service cloud-init-hotplugd describe
 * No description for cloud-init-hotplugd
 * healthcheck: no description
 * unhealthy: no description
 * cgrp_cleanup: Kill all processes in the cgroup
```
I propose we minimally:
- provide descriptions of this service here.
- it would be nice to have an easy comment  header providing context

do we want or need a checkconfig function?

Reply:
I based this openrc file on the other existing cloud-init openrc files so I configured it accordingly.

Whereas if you look at the hotplug file I've packaged for Alpine for some months it does include a description: https://git.alpinelinux.org/aports/tree/community/cloud-init/cloud-init-hotplugd.initd

So you're flagging a more general issue about openrc files in cloud-init in general.

Reply:
Ahh, ok. Looking through those openrc files now, it is a general question/concern about the preexisting gentoo openrc files and doesn't really need to be handled in this PR, but setting a good example for docs in this supplemental addtion would be helpful and we'll reflect it elsewhere as time permits.

Reply:
I have tidied up the cloud-init-hotplug openrc script - it now has a description and switch from a start_pre function to a start function.

I'll look at making changes to the other openrc script via a separate PR.